hack/update-codegen.sh

```diff
@@ -24,9 +24,64 @@ CONTAINERIZE_IMAGE=golang:1.18.4 containerize ./hack/update-codegen.sh
 echodate "Running go generate"
 go generate ./pkg/...
 
+CRD_DIR=pkg/crd/k8c.io
+
 echodate "Generating openAPI v3 CRDs"
 go run sigs.k8s.io/controller-tools/cmd/controller-gen \
   crd \
   object:headerFile=./hack/boilerplate/ce/boilerplate.go.txt \
   paths=./pkg/apis/... \
-  output:crd:dir=./pkg/crd/k8c.io
+  output:crd:dir=./$CRD_DIR
+
+annotation="kubermatic.k8c.io/location"
+locationMap='{
+  "applicationdefinitions.apps.kubermatic.k8c.io": "master",
```

Comment:
```suggestion
  "applicationdefinitions.apps.kubermatic.k8c.io": "master,seed",
```

applicationDefinition CRs are replicated from master to seed so we must install applicationDefinition CRD into seeds.